### ISSUE TITLE
Need emscripten demonstration in GitHub Actions

### ISSUE THREAD

**Issue by kulp** [kulp/tenyr#75]
In the Travis-CI flow (removed in #72), emscripten support was demonstrated, but in the migration to GitHub Actions (begun in #66), emscripten is not yet represented.

We could use [`container`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer) in order to set up Docker to load emscripten, but it might be possible to avoid Docker entirely by using https://github.com/mymindstorm/setup-emsdk instead.

**Comment by kulp** [comment]
This issue seems to be a duplicate of #91, which itself is being deferred until after #93.
